order/src/lib.rs

```diff
@@ -0,0 +1,214 @@
+//! Contains the order type as described by the specification with serialization as described by the openapi documentation.
+//!
+//! This is in its own crate because we want to share this module between the orderbook and the solver.
+
+use chrono::{offset::Utc, DateTime};
+use primitive_types::H160;
+use serde::{de, Deserialize, Serialize};
+use std::fmt;
+
+#[derive(Eq, PartialEq, Clone, Copy, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "lowercase")]
+pub enum FillType {
+    FillKill,
```

Comment:
No strong opinion, but I would call it `FillOrKill`

Reply:
Will rename it here and also in the openapi (where it was `fillkill`).